units_cpp/tech.h

```diff
@@ -1,26 +1,31 @@
+/* Contains the class Tech, which stores a collection of techs. */
+
 #pragma once
+
 #include <iostream>
+
 #include "colour.h"
 #include "json.h"
 
 using json = nlohmann::json;
 
+
 class Tech {
- public:
-  int red;
-  int green;
-  int blue;
+	public:
+		int red;
+		int green;
+		int blue;
 
-  Tech();
-  Tech(int red, int green, int blue);
-  void add(const Tech &t);
-  void max(const Tech &t);
-  void increment(Colour colour);
-  bool includes(const Tech &t) const;
+		Tech();
+		Tech(int red, int green, int blue);
+		void add(const Tech &t);
+		void max(const Tech &t);  // Sets all values of this tech to the max of (this tech, the given tech)
+		void increment(Colour colour);  // Increment this colour by one. Will not increment BLACK.
+		bool includes(const Tech &t) const;  // Check if this tech is a proper superset of the given one.
```

Comment:
You probably just mean "superset". "proper superset" means "superset AND not equal to".